### PR TITLE
Corrected the license to be LogRhythm MIT and adjusted the README

### PR DESCRIPTION
**Pull request by KjellKod**
This pull request corrects the LogRhythm LICENSE and improves the Readme.

The README can best be read at: https://github.com/KjellKod/StopWatch-1/blob/LogRhythm_OpenSource_Compliance/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,24 @@
+
+AlarmClock
+==========
+This gives the option to query whether a specified time period has expired. The utility is made to have a very low CPU clock cycle cost for the client compared to using StopWatch or std::chrono.
+
+The implementation of AlarmClock was made to be  as accurate as possible, with the assumption that low CPU usage and cheap, CPU wise,  API calls are more important than absolute accuracy in terms of some extra microseconds of expiration overhead. 
+
+Common expiration time overhead ranged from 50 - 150 microseconds on the current testing platform when the Alarm clock was checked for expiration in a continous loop. (which normally is not a good usage pattern)
```
Comment (Schatzman) [inline README.md]: Continuous is misspelled in the last sentence. Also, the additional fragment in parens should come before the period in that last sentence.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,24 @@
+
+AlarmClock
+==========
+This gives the option to query whether a specified time period has expired. The utility is made to have a very low CPU clock cycle cost for the client compared to using StopWatch or std::chrono.
+
+The implementation of AlarmClock was made to be  as accurate as possible, with the assumption that low CPU usage and cheap, CPU wise,  API calls are more important than absolute accuracy in terms of some extra microseconds of expiration overhead.
```
Comment (Schatzman) [inline README.md]: CPU-wise

File: README.md (hunk 1 above)
Comment (KjellKod) [inline README.md]: got it.

File: README.md (hunk 2 above)
Comment (KjellKod) [inline README.md]: done

**Comment by Schatzman** [comment]
:+1:

File: README.md [hunk 3]
```diff
@@ -1,4 +1,26 @@
+
+AlarmClock
+==========
+This gives the option to query whether a specified time period has expired. The utility is made to have a very low CPU clock cycle cost for the client compared to using StopWatch or std::chrono.
+
+The implementation of AlarmClock was made to be  as accurate as possible, with the assumption that low CPU usage and cheap, CPU-wise,  API calls are more important than absolute accuracy in terms of some extra microseconds of expiration overhead.
```
Comment (john-gress) [inline README.md]: "with the assumption that low CPU usage and cheap, CPU-wise, API calls ..." doesn't sound right. Do you mean, "with the tradeoff of less CPU usage resulting in a less accurate expiration capability"?

File: README.md [hunk 4]
```diff
@@ -1,4 +1,26 @@
+
+AlarmClock
+==========
+This gives the option to query whether a specified time period has expired. The utility is made to have a very low CPU clock cycle cost for the client compared to using StopWatch or std::chrono.
+
+The implementation of AlarmClock was made to be  as accurate as possible, with the assumption that low CPU usage and cheap, CPU-wise,  API calls are more important than absolute accuracy in terms of some extra microseconds of expiration overhead. 
+
+Common expiration time overhead ranged from 50 - 150 microseconds on the current testing platform when the Alarm clock was checked for expiration in a continuous loop (which normally is not a good usage pattern).
```
Comment (john-gress) [inline README.md]: "ranged" -> ranges

File: README.md (hunk 3 above)
Comment (john-gress) [inline README.md]: In multiple places, I see two spaces being used when you only need one.

Comment (KjellKod) [inline README.md]: thanks. I'm updating it.

Comment (KjellKod) [inline README.md]: fixed

Comment (KjellKod) [inline README.md]:
```
The implementation of AlarmClock was made to be as accurate and efficient as possible. The focus is on low CPU consumption, with the tradeoff of less CPU usage resulting in a less accurate expiration capability. 
```

File: README.md (hunk 4 above)
Comment (KjellKod) [inline README.md]: fixed

**Comment by Schatzman** [comment]
:+1:

**Comment by john-gress** [comment]
👍

**Pull request merged**